### PR TITLE
[IOTDB-3420] [TO rel/0.13] fix show paths set schema template t1 error

### PR DESCRIPTION
**Pull request by RYH61**
see detail: https://issues.apache.org/jira/browse/IOTDB-3420

after:
IoTDB> create timeseries root.sg.device1.t with datatype=BOOLEAN,encoding=PLAIN,compression=UNCOMPRESSED;
Msg: The statement is executed successfully.
IoTDB> create timeseries root.sg.device2.t1 with datatype=BOOLEAN,encoding=PLAIN,compression=UNCOMPRESSED;
Msg: The statement is executed successfully.
IoTDB> create schema template t1 (temperature FLOAT encoding=RLE, status BOOLEAN encoding=PLAIN compression=SNAPPY)
Msg: The statement is executed successfully.
IoTDB> set schema template t1 to root.sg.device1
Msg: The statement is executed successfully.
IoTDB> show paths set schema template t1
+----------------+
 |    child paths      |
+----------------+
 |root.sg.device1|
+----------------+
Total line number = 1
It costs 0.065s

图片

### CODE REVIEW

**Review by bigreybear — APPROVED** [review]
LGTM

File: server/src/main/java/org/apache/iotdb/db/metadata/mtree/MTree.java [hunk 1]
```diff
@@ -1783,6 +1783,10 @@ protected boolean processFullMatchedMNode(IMNode node, int idx, int level)
                 return true;
               }
 
+              if (node.isMeasurement()) {
+                return false;
```
Comment (bigreybear) [inline server/src/main/java/org/apache/iotdb/db/metadata/mtree/MTree.java:1787]: Excellent correction! While `return false` should have fixed this issue, but `return true` might be a clearer statement that the `traverser` should not dive into this branch any more.

Comment (RYH61) [reply server/src/main/java/org/apache/iotdb/db/metadata/mtree/MTree.java]: You are righ

Comment (RYH61) [reply server/src/main/java/org/apache/iotdb/db/metadata/mtree/MTree.java]: fixed~

**Review by mychaow — APPROVED** [review]
LGTM

**Pull request merged**